Assets/_nathan/materials/outdoors.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: bef1f7425e80cd242bb3bea12e1f108d
+guid: 3a013eff9f80d304d9db718d4bf4b72d
```

Comment:
Another broken meta file. Again, should be fixed when I .gitignore .meta files in the assets folder. Won't comment on others like this. Will ping you when I update .gitignore